### DISCUSSION TITLE
Audacious is copying my music???

### DISCUSSION THREAD

**Discussion by tobeypeters** [audacious-media-player/audacious#123] [General]
I noticed that there was music files in my Home folder and I couldn't figure out how they got there.  But, I noticed that any song I play is being copied into my Home folder.  How do I stop that?

**Comment by mschwendt** [comment]
The most likely cause is that you've accidentally enabled the recording feature within the settings "Audio > Recording Settings".

**Reply by mschwendt** [reply]
Oh, and to expand on that. If it's enabled, you get a circular "Record Stream" button next to the right "Next" track button. Activating that accidentally has happened before, afaik.
